src/index.js

```diff
@@ -46,35 +51,27 @@ const onLoaded = () => {
   handleRouting();
 };
 
-const renderLevelComplete = root => {
-  const completeHTML = `
-  <div id="complete">
-    <h1>Level Complete!</h1>
-      <div id="controls">
-        <button title="Home" id="change-level"><span class="material-symbols-outlined">home</span></button>
-    </div>
-  `;
-  root.innerHTML = completeHTML;
-  document
-    .querySelector("#change-level")
-    .addEventListener("click", () => {
-      location.hash = "";
-     renderMenu(root);
-    })
-  ;
-};
-
 const renderLevel = (root, levelNumber) => {
   const gameHTML = `
+  <div id="overlay-screen"></div>
   <div id="game" class="hide">
     <div id="controls">
       <button title="Undo (z key)" id="undo"><span class="material-symbols-outlined">undo</span></button>
-      <button title="Redo (r key)" id="redo" disabled><span class="material-symbols-outlined">redo</span></button>
+      <button title="Reset (r key)" id="reset"><span class="material-symbols-outlined">refresh</span></button>
```

Comment:
Shorten HTML lines to 70 characters max.

Reply:
These are still long -- I'd move the `<span>`s on to their own lines within the buttons